pkg/cmd/auth/shared/login_flow.go

```diff
@@ -231,3 +232,34 @@ func sshKeyUpload(httpClient *http.Client, hostname, keyFile string, title strin
 
 	return add.SSHKeyUpload(httpClient, hostname, f, title)
 }
+
+func getCurrentLogin(httpClient httpClient, hostname, authToken string) (string, error) {
```

Comment:
This is more or less equivalent to what happens in `api.CurrentLoginName` which is why no test changes were necessary.